examples/client/Locomotion/src/Components/PhoneNumberInput/index.js

```diff
@@ -1,27 +1,42 @@
 import React, { useState } from 'react';
 import PhoneInput from 'react-native-phone-number-input';
+import { AsYouType } from 'libphonenumber-js';
 import i18n from '../../I18n';
 import { ERROR_COLOR } from '../../context/theme';
 import codes from './codes.json';
 
-
 const PhoneNumberInput = ({
-  onPhoneNumberChange, defaultCode, autoFocus, error, value,
+  onPhoneNumberChange,
+  defaultCode,
+  autoFocus,
+  error,
+  value,
 }) => {
   const [isFocused, setIsFocused] = useState(false);
   const initialCode = codes.find(v => v.code === defaultCode);
   const [countryCode, setCountryCode] = useState(initialCode.dialCode);
+  const asYouTypePhoneNUmber = new AsYouType();
```

Comment:
@dorshay6 Tiny, tiny, tiny nitpick: this Variable is `asYouTypePhoneNUmber` should be `asYouTypePhoneNumber`